src/software/simulation/er_force_simulator.h

```diff
@@ -109,66 +109,46 @@ class ErForceSimulator : public QObject
     Timestamp getTimestamp() const;
 
     /**
-     * Resets the current firmware time to 0
+     * Resets the current time to 0
      */
-    static void resetCurrentFirmwareTime();
+    void resetCurrentTime();
 
    private:
-    /**
-     * Get the current time.
-     *
-     * This is passed into a `FirmwareWorld`, which requires that it is static (as it
-     * is C code). This will just return `current_firmware_time`, which should be updated
-     * to the actual current time before ticking any firmware.
-     *
-     * @return The value of `current_firmware_time`, in seconds.
-     */
-    static float getCurrentFirmwareTimeSeconds();
-
     /**
      * Sets the primitive being simulated by the robot in simulation
      *
      * @param id The id of the robot to set the primitive for
      * @param primitive_msg The primitive to run on the robot
      * @param simulator_robots The robots to set the primitives on
-     * @param simulator_ball The simulator ball to use in the primitives
      * @param vision_msg The vision message
      */
     static void setRobotPrimitive(
-        RobotId id, const TbotsProto_Primitive& primitive_msg,
-        std::map<std::shared_ptr<ErForceSimulatorRobot>,
-                 std::shared_ptr<FirmwareWorld_t>>& simulator_robots,
-        std::shared_ptr<ErForceSimulatorBall> simulator_ball,
+        RobotId id, const TbotsProto::Primitive& primitive_msg,
+        std::vector<std::shared_ptr<ErForceSimulatorRobot>>& simulator_robots,
         const TbotsProto::Vision& vision_msg);
 
     /**
      * Update Simulator Robot and get the latest robot control
      *
      * @param handle_robot_log_proto Function pointer to handle robot log proto
-     * @param simulator_robots Map of simulator robots
+     * @param simulator_robots Vector of simulator robots
```

Comment:
please update the javadocs

Reply:
hmm I don't see this, might be a previous commit?

Reply:
there's no `handle_robot_log_proto` argument anymore

Reply:
ah, removed